src/lib.rs

```diff
@@ -78,23 +92,39 @@ pin_project! {
 /// Builder for `TlsListener`.
 #[derive(Clone)]
 pub struct Builder {
-    server_config: Arc<ServerConfig>,
+    acceptor: TlsAcceptor,
```

Comment:
Unfortunately native_tls doesn't have an infallible equivalent to ServerConfig, so for simplicity and consistency it made sense just to make the user provide a TlsAcceptor in both instances

Reply:
I have changed the functions to accept `T: Into<TlsAcceptor>`, such that a user can pass in any of `native_tls::TlsAcceptor` or `tokio_native_tls::TlsAcceptor`, `tokio_rustls::TlsAcceptor` or `Arc<ServerConfig>`.

It remains a very minor breaking change but it is documented in the CHANGELOG now.